core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -293,6 +296,24 @@ public HttpResponse doCreateAdminUser(StaplerRequest req, StaplerResponse rsp) t
             }
         }
     }
+    
+    @RequirePOST
+    public HttpResponse doConfigureRootUrl(StaplerRequest req, @QueryParameter String rootUrl) {
```

Comment:
Probably should add @Restricted(NoExternalUse.class) since it is a new public method.